src/app-layout/__integ__/app-layout-sticky-elements.test.ts

```diff
@@ -67,7 +68,7 @@ test(
       await page.windowScrollTo({ top: 200 });
       const { bottom: pageHeaderBottom } = await page.getBoundingBox('header');
       const { top: tableHeaderTop } = await page.getBoundingBox(page.findStickyTableHeader().toSelector());
-      expect(tableHeaderTop).toEqual(pageHeaderBottom - 1);
+      expect(tableHeaderTop).toEqual(pageHeaderBottom - CONTAINER_ROOT_BORDER);
```

Comment:
Can't we just remove this since we subtract 0?

Reply:
We can now. Originally was there because we had [this code](https://github.com/cloudscape-design/components/blob/40a405bd591436179a2e1e9a4287664bc7f6582c/src/container/use-sticky-header.ts#L25-L33) that checks for the container and header borders and accounts for it in sticky header. Removed both in next revision because borders no longer add to container height.